questionnaire/questionnaire-service.test.js

```diff
@@ -1,7 +1,27 @@
+/* eslint-disable no-shadow */
```

Comment:
Are there Supertest routes test for the Express routing that implements these functions?